app/actions/get_info.py

```diff
@@ -0,0 +1,16 @@
+from app.add_some_abilities.add_protection import protection
+from app.add_some_abilities.add_power import power
+from app.add_some_abilities.add_potion_abilities import add_ability_from_potion
+
+
+def get_knights_info(knightscfg: dict) -> list:
+    knights = []
+    for knight in knightscfg:
+        knights.append(knightscfg[knight])

```

Comment:
Consider using a more descriptive variable name than 'knight'. It would improve readability. For example, 'knight_name' or 'knight_key' could be more descriptive.